app/views/sources/_image.html.erb

```diff
@@ -1,20 +1,15 @@
+<% content_for :head_script do %>
+<% javascript_include_tag 'openseadragon' %>
+<% end %>
+
 <article class='image'>
   <div class='media-outer-container image'>
     <div class='media-inner-container'>
 
-      <a class="lightboxlink"
-         data-lightbox="image-1"
-         data-title="<%= @source.main_asset.alt_text %>"
-         href="<%= base_src + @source.main_asset.file_name %>">
-
-        <div style="position: relative;">
-            <div style="position: absolute; top: 0; left: 0; width: 700px; height: 485px; z-index: 2;"></div>
-            <img src="<%= base_src + @source.main_asset.file_name %>"
-                 alt="<%= @source.main_asset.alt_text %>"
-                 style="position: absolute; top: 0; left: 0; z-index: 1; max-width: 700px; max-height: 485px;" />
-        </div>
+      <div id="osd-viewer" oncontextmenu="return false;"></div>
+      <p id="image-caption"><%= html_escape_once(@source.main_asset.alt_text) %></p>
```

Comment:
From what I've quickly read, it's supposed to HTML-escape a string, but should skip character sequences that are already encoded.  For example, if someone is being diligent and entering `this &amp; that`, it will not further encode the ampersand in that string.